tests/Main.hs

```diff
@@ -193,7 +193,19 @@ runClashTest = defaultMain $ clashTestRoot
       , clashTestGroup "Cores"
         [ clashTestGroup "Xilinx"
           [ clashTestGroup "VIO"
-            [ runTest "OutputBusWidthExceeded" def{
+            [ runTest "DuplicateOutputNames" def{
+                hdlTargets=[VHDL]
```

Comment:
Any particular reason you just test `VHDL`?

Reply:
Yes! The Cores testsuite is getting quite slow because of Vivado. This particular test doesn't care about which HDL it does - name generation / collision detection is the same across all HDLs.